test/gui/shared/scripts/pageObjects/Activity.py

```diff
@@ -0,0 +1,30 @@
+import names
+import squish
+from objectmaphelper import RegularExpression
+from helpers.FilesHelper import buildConflictedRegex
+
+
+class Activity:
+    SUBTAB = {
```

Comment:
Can you rename it to something else. It is a bit hard to understand which tab/button is this.

Reply:
The name of the tab depends upon the tab name passed here `squish.clickTab(squish.waitForObject(self.SUBTAB), tabName)` So, it is like a generic selector, not a specific one.

Reply:
The name of the tab depends upon the tab name passed here squish.clickTab(squish.waitForObject(self.SUBTAB), tabName) So, it is like a generic selector, not a specific one.